volumes/miovision/update_intersections/Readme.md

```diff
@@ -1,7 +1,401 @@
+
+    ```sql
+    WITH movements AS (
+        SELECT
+            intersection_uid, leg,
+            movement_uid || ' - ' || movement_pretty_name || ' (' ||
+                    string_agg(classification_uid || ' - ' || classification, ', '::text ORDER BY classification_uid) || ')'               
+                AS mvmts
+        FROM miovision_api.intersection_movements
+        LEFT JOIN miovision_api.classifications USING (classification_uid)
+        LEFT JOIN miovision_api.movements USING (movement_uid)
+        WHERE intersection_uid IN (66) --adjust uid here
+            AND classification_uid NOT IN (3, 4, 5, 8, 9) --since these just mirror lights
+        GROUP BY
+            intersection_uid,
+            leg,
+            movement_uid,
+            movement_pretty_name
+    )
+
+    SELECT
+        intersection_uid,
+        leg,
+        string_agg(mvmts, chr(10) ORDER BY mvmts) AS movements
+    FROM movements
+    GROUP BY
+        intersection_uid,
+        leg
+    ORDER BY
+        intersection_uid,
+        leg
+    ```
+
+    e) **Manual edits**  
+    If you find you need to manually add movements to the above,
+	download the output of the query into a CSV, manually edit the CSV, then
+	append it to `miovision_api.intersection_movements` by modifying the below python snippet, (or use an SQL INSERT statement):
+
+	```python
+	import pandas as pd
+	import psycopg2
+	from psycopg2.extras import execute_values
+
+	import configparser
+	import pathlib
+
+	# Insert code to read configuration settings.
+	postgres_settings = {your_postgres_config}
+
+	# Insert the name of your CSV file.
+	df = pd.read_csv({your_file.csv})
+	df_list = [list(row.values) for i, row in df.iterrows()]
+
+	with psycopg2.connect(**postgres_settings) as conn:
+		with conn.cursor() as cur:
+			insert_data = """INSERT INTO miovision_api.intersection_movements(intersection_uid, classification_uid, leg, movement_uid) VALUES %s"""
+			execute_values(cur, insert_data, df_list)
+			if conn.notices != []:
+				print(conn.notices)
+	```
+
+## Backfill/Aggregate new intersection data
+
+3. Now that the intersection is configured and the raw volumes data is in the database, we have to finish aggregating the data.
+
+	a) **Backfill `miovision_api.volumes`**   
+    If not already complete, use the [api script](../api/intersection_tmc.py) with `--pull` to backfill `miovision_api.volumes` table between the date_installed and previous day. 
+
+	b) **Backfill additional tables**  
+    We now have to run a couple of functions manually with (%s::date, %s::date) being (start_date::date, end_date::date) to finish aggregating the backfilled data.  
+	```sql
+	SELECT miovision_api.find_gaps(%s::date, %s::date);
+	SELECT miovision_api.aggregate_15_min_tmc(%s::date, %s::date);
+	SELECT miovision_api.aggregate_15_min(%s::date, %s::date); 
+	SELECT miovision_api.report_dates(%s::date, %s::date);
```

Comment:
Looking through #429 I realize some of these functions need updates to be used for backfilling. 
Either an option to run for only one intersection_uid like [Jedwin proposed in 2022](https://github.com/CityofToronto/bdit_data-sources/blob/fbfa27a51f500dade17840520e736b434f6066f6/volumes/miovision/sql/function-aggregate-volumes_15min_mvt_filter_65_66.sql): 
OR other updates so they don't insert duplicate data.
@radumas how would you recommend proceeding?